src/tests/task1.spec.js

```diff
@@ -0,0 +1,37 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+
+test.describe('filtering checks', () => {
+    test.beforeEach(async ({ loginPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+    });
+
+    test('Perform and verify sorting on the Inventory page (az)', async ({ inventoryPage }) => {
```

Comment:
Well done!
Now let's make code clearer and use [parameterized tests](https://playwright.dev/docs/test-parameterize)